### ISSUE TITLE
xinference环境无法加载openai sdk产生临时图片链接

### ISSUE THREAD

**Issue by weiwuxian1998** [xorbitsai/inference#2084]
### System Info / 系統信息

(Xinference) root@Server-058266b4-896f-4df5-9763-65d3e241d655:~# pip list
Package                           Version
--------------------------------- ------------
accelerate                        0.31.0
addict                            2.4.0
aiobotocore                       2.7.0
aiofiles                          23.2.1
aiohttp                           3.9.5
aioitertools                      0.11.0
aioprometheus                     23.12.0
aiosignal                         1.3.1
aliyun-python-sdk-core            2.15.1
aliyun-python-sdk-kms             2.16.3
altair                            5.3.0
annotated-types                   0.7.0
anthropic                         0.30.0
antlr4-python3-runtime            4.9.3
anyio                             4.4.0
argcomplete                       3.4.0
async-timeout                     4.0.3
attrdict                          2.0.1
attrs                             23.2.0
audioread                         3.0.1
auto_gptq                         0.7.1
autoawq                           0.2.5
autoawq_kernels                   0.0.6
bcrypt                            4.1.3
bitsandbytes                      0.43.1
boto3                             1.28.64
botocore                          1.31.64
cdifflib                          1.2.6
certifi                           2024.6.2
cffi                              1.16.0
charset-normalizer                3.3.2
chatglm-cpp                       0.4.0
click                             8.1.7
cloudpickle                       3.0.0
cmake                             3.29.6
colorama                          0.4.6
coloredlogs                       15.0.1
contourpy                         1.2.1
controlnet_aux                    0.0.7
crcmod                            1.7
cryptography                      42.0.8
cycler                            0.12.1
Cython                            3.0.10
datasets                          2.18.0
decorator                         5.1.1
diffusers                         0.29.2
dill                              0.3.8
diskcache                         5.6.3
distro                            1.9.0
ecdsa                             0.19.0
editdistance                      0.8.1
einops                            0.8.0
einx                              0.3.0
encodec                           0.1.1
fastapi                           0.110.3
ffmpy                             0.3.2
filelock                          3.15.4
FlagEmbedding                     1.2.10
fonttools                         4.53.0
frozendict                        2.4.4
frozenlist                        1.4.1
fsspec                            2023.10.0
gast                              0.6.0
gekko                             1.1.3
gradio                            4.26.0
gradio_client                     0.15.1
h11                               0.14.0
hf_transfer                       0.1.6
hiredis                           2.3.2
httpcore                          1.0.5
httptools                         0.6.1
httpx                             0.27.0
huggingface-hub                   0.23.4
humanfriendly                     10.0
idna                              3.7
imageio                           2.34.2
importlib_metadata                8.0.0
importlib_resources               6.4.0
inflect                           7.3.0
interegular                       0.3.3
Jinja2                            3.1.4
jiter                             0.5.0
jmespath                          0.10.0
joblib                            1.4.2
jsonschema                        4.22.0
jsonschema-specifications         2023.12.1
kiwisolver                        1.4.5
lark                              1.1.9
lazy_loader                       0.4
libnacl                           2.1.0
librosa                           0.10.2.post1
litellm                           1.41.2
llama_cpp_python                  0.2.79
llvmlite                          0.43.0
lm-format-enforcer                0.10.1
markdown-it-py                    3.0.0
MarkupSafe                        2.1.5
matplotlib                        3.9.0
mdurl                             0.1.2
modelscope                        1.15.0
more-itertools                    10.3.0
mpmath                            1.3.0
msgpack                           1.0.8
multidict                         6.0.5
multiprocess                      0.70.16
nemo_text_processing              1.0.2
nest-asyncio                      1.6.0
networkx                          3.3
ninja                             1.11.1.1
numba                             0.60.0
numpy                             1.26.4
nvidia-cublas-cu12                12.1.3.1
nvidia-cuda-cupti-cu12            12.1.105
nvidia-cuda-nvrtc-cu12            12.1.105
nvidia-cuda-runtime-cu12          12.1.105
nvidia-cudnn-cu12                 8.9.2.26
nvidia-cufft-cu12                 11.0.2.54
nvidia-curand-cu12                10.3.2.106
nvidia-cusolver-cu12              11.4.5.107
nvidia-cusparse-cu12              12.1.0.106
nvidia-ml-py                      12.555.43
nvidia-nccl-cu12                  2.20.5
nvidia-nvjitlink-cu12             12.5.40
nvidia-nvtx-cu12                  12.1.105
omegaconf                         2.3.0
openai                            1.35.7
opencv-contrib-python             4.10.0.84
opencv-python                     4.10.0.84
optimum                           1.20.0
orjson                            3.10.5
oss2                              2.18.6
outlines                          0.0.34
packaging                         24.1
pandas                            2.2.2
passlib                           1.7.4
peft                              0.11.1
pillow                            10.3.0
pip                               24.0
pipx                              1.6.0
platformdirs                      4.2.2
plumbum                           1.8.3
pooch                             1.8.2
prometheus_client                 0.20.0
prometheus-fastapi-instrumentator 7.0.0
protobuf                          5.27.2
psutil                            6.0.0
py-cpuinfo                        9.0.0
pyarrow                           16.1.0
pyarrow-hotfix                    0.6
pyasn1                            0.6.0
pycparser                         2.22
pycryptodome                      3.20.0
pydantic                          2.7.4
pydantic_core                     2.18.4
pydub                             0.25.1
Pygments                          2.18.0
pynini                            2.1.5
pynvml                            11.5.0
pyparsing                         3.1.2
python-dateutil                   2.9.0.post0
python-dotenv                     1.0.1
python-jose                       3.3.0
python-multipart                  0.0.9
pytz                              2024.1
PyYAML                            6.0.1
pyzmq                             26.0.3
quantile-python                   1.1
ray                               2.31.0
redis                             5.0.7
referencing                       0.35.1
regex                             2024.5.15
requests                          2.32.3
rich                              13.7.1
rouge                             1.0.1
rpds-py                           0.18.1
rpyc                              6.0.0
rsa                               4.9
ruff                              0.5.0
s3fs                              2023.10.0
s3transfer                        0.7.0
sacremoses                        0.1.1
safetensors                       0.4.3
scikit-image                      0.24.0
scikit-learn                      1.5.0
scipy                             1.14.0
semantic-version                  2.10.0
sentence-transformers             3.0.1
sentencepiece                     0.2.0
setuptools                        69.5.1
sglang                            0.1.17
shellingham                       1.5.4
simplejson                        3.19.2
six                               1.16.0
sniffio                           1.3.1
sortedcontainers                  2.4.0
soundfile                         0.12.1
soxr                              0.3.7
sse-starlette                     2.1.2
starlette                         0.37.2
sympy                             1.12.1
tabulate                          0.9.0
tblib                             3.0.0
tensorizer                        2.9.0
threadpoolctl                     3.5.0
tifffile                          2024.6.18
tiktoken                          0.7.0
timm                              1.0.7
tokenizers                        0.19.1
tomli                             2.0.1
tomlkit                           0.12.0
toolz                             0.12.1
torch                             2.3.0
torchaudio                        2.3.0
torchvision                       0.18.0
tqdm                              4.66.4
transformers                      4.40.2
transformers-stream-generator     0.0.5
triton                            2.3.0
typeguard                         4.3.0
typer                             0.11.1
typing_extensions                 4.12.2
tzdata                            2024.1
urllib3                           2.0.7
userpath                          1.9.2
uvicorn                           0.30.1
uvloop                            0.19.0
vector-quantize-pytorch           1.14.44
vllm                              0.4.3
vllm-flash-attn                   2.5.8.post2
vocos                             0.1.0
watchfiles                        0.22.0
websockets                        11.0.3
WeTextProcessing                  1.0.2
wget                              3.2
wheel                             0.43.0
wrapt                             1.16.0
xformers                          0.0.26.post1
xinference                        0.12.3
xoscar                            0.3.0
xxhash                            3.4.1
yapf                              0.40.2
yarl                              1.9.4
zipp                              3.19.2
zmq                               0.0.0
zstandard                         0.22.0

### Running Xinference with Docker? / 是否使用 Docker 运行 Xinfernece？

- [ ] docker / docker
- [X] pip install / 通过 pip install 安装
- [ ] installation from source / 从源码安装

### Version info / 版本信息

0.12.3

### The command used to start Xinference / 用以启动 xinference 的命令

XINFERENCE_HOME=/root/.xinference/ XINFERENCE_MODEL_SRC=modelscope xinference-local --host 0.0.0.0 --port 8866 

### Reproduction / 复现过程

Langchain3图文对话上传了一张图片，通过调用openAI sdk产生了临时连接， 在xinference的环境下无法打开，换用其他环境正常打开

### Expected behavior / 期待表现

请问这种情况是否xinference有关

**Comment by weiwuxian1998** [comment]
已解决，版本问题，更新到最新版的xinference
